### ISSUE TITLE
Stop lying to us (behavior score)

### ISSUE THREAD

**Issue by vanndano** [ValveSoftware/Dota2-Gameplay#22253]
### Description

Gettign punished for "overwatch penalties" when there has been no actual time to review. This I beleive is a false positive of whatever the hell the devs are doing to auto send palyers to LP

15 positive matches 2 conducts in a row, still somehow on a game where there are 2 griefers IM THE ONE THAT ENDS UP PUNISHED. Screenshots below

Match ID 7987252157

Clearly axe offlane and mid alchemist wanted nothing to do with the game past minute 5, went out of their way to avoid fights and jungled after losing their lanes. they went 0-5 0-2. I reported both of them 

To my surprise IM THE ONE THAT ENDED UP IN LP, all in one match without any previous reports. 

Seriously fix your dumb system, Im trying to recalibrate and every game is a skill imbalanced shitshow. Someone always gets blamed

But banning me for 24 hours cuz a weekend player decided it would be cute to report me is unacceptable despite that me and the other support were the ones still trying.

Worse than that, lying to the player and saying this is "an overwatch conviction" and not a >report threshold Is unacceptable. 

That happening when I have 0 record of reports or abandons in the past 30 matches is unacceptable. 

At some point you have to realize many factors affect reports. If its a loss, more people report, if its a weekend, more people report, if the person uses chat and doesnt grief, he is more likely to get reported wrongly. But the insulting part is the actual griefers didnt get punished. 

Enough is enough, Valve fix your shit, you've had 12 years. 

### Screenshots

![image](https://github.com/user-attachments/assets/cd82e54a-b71a-41b4-9de5-c3213b97103e)

![image](https://github.com/user-attachments/assets/a206afb5-59dc-439c-8595-a8230b8e6ce2)

![image](https://github.com/user-attachments/assets/1525be25-3c0c-4d69-80a8-80bfe136932b)

![image](https://github.com/user-attachments/assets/8292063c-7fe6-410e-8a6c-3bb8141b0552)

![image](https://github.com/user-attachments/assets/c7858ca1-41e4-49fa-9299-48b47aabbaed)

**Comment by vanndano** [comment]
#21667

**Comment by el1diab1lo8** [comment]
Your automatic system throws players into LP just like that, your online has dropped a lot and it is not because of the new product, your game has almost nothing good left, only shortcomings

**Comment by el1diab1lo8** [comment]
7985925992  I got into LP because my teammates decided to lose the game but made me the scapegoat. Is this adequate or is the game not for adequate players?

**Comment by el1diab1lo8** [comment]
In your game every day there is less and less online and there will be no more the less people the more you wait search for the game the more you wait the less people there will be

**Comment by alar1ch** [comment]
14,275 matches in dota 49.76% winrate you legend rank are just a terrible player,for 14k matches you still buying tranquil for silencer idk just stop playing

**Comment by vanndano** [comment]
> 14,275 matches in dota 49.76% winrate you legend rank are just a terrible player,for 14k matches you still buying Tranquil Boots for Silencer idk just stop playing

Right, the pos 5 tranquils is why alchemist afked the whole game and axe also did. very insightful, if ur not gonna bother to read dont comment
